### PR TITLE
Style/jasons style branch

### PR DESCRIPTION
**Pull request by jasonkwm**
## General Changes & Checklist

- [x] Get PAW balance from smart contract
- [x] Change all number to BigNumberish
- [x] General Styling of Manage Page
- [x] Convert Address to Coin & Link Token To Svg
- [ ] get USD amount for coin through oracle
- [x] QuickAction Panel Add Approval feature
- [ ] Page refresh on wallet change
- [x] Page refresh on success
- [ ] get expired locked paw (backend function empty)
- [x] Improve UI/UX for Success or Error (replace prompt with Modal)
- [x] Dark Theme Compatibility 
- [x] Mobile View
- [x] Translate Manage page 
- [x] Added temporary SVG for PAW Coin icon

## Developer Notes

Add any notes here that may be helpful for reviewers.

### CODE REVIEW

File: src/ui-config/marketsConfig.tsx
Comment (neosizzle) [inline src/ui-config/marketsConfig.tsx:1]: Looks like you are hardcoding the address for this variable, Ill send you the address soon so you can add it to your .env

File: src/locales/en/messages.js
Comment (neosizzle) [inline src/locales/en/messages.js:1]: Locales not functioning in build,
image

Comment (neosizzle) [reply src/locales/en/messages.js:1]: This can be fixed by rebuilding with different i18n parameters, can be tested by building and serving locally.

**Review by neosizzle — CHANGES_REQUESTED** [review]

File: src/locales/en/messages.js
Comment (jasonkwm) [reply src/locales/en/messages.js:1]: i think its alright now. :D

**Comment by jasonkwm** [comment]
Ok i hope the problem is fix. please review again. :D

**Review by neosizzle — CHANGES_REQUESTED** [review]
image
Manage Paw left panel infinite loading, I think its something to do with this error :


image


Where you called a smart contract without declaring a signer. Everything else is OK for me.

**Comment by neosizzle** [comment]
The above error turns out to be a misconfiguration in vercel, it is already rectified on that side.

**Review by neosizzle — APPROVED** [review]

**Pull request merged**